composer.json

```diff
@@ -24,6 +24,7 @@
         "imagine/imagine": "~0.6.1",
         "liip/theme-bundle": "1.3.*",
         "massive/search-bundle": "dev-develop",
+        "massive/search-bundle": "dev-feature/index-name-evaluation",
```

Comment:
don`t forget to remove this after merging https://github.com/massiveart/MassiveSearchBundle/pull/65!